docs/userguide/parameters.md

```diff
@@ -1,4 +1,205 @@
+above would imply that "Solvent A" is more similar to "Solvent B" than to "Solvent C" 
+because the number 1 is closer to 2 than to 3. 
+This implied ordering is however not generally the case for the provided labels.
+In general, it is not even possible to describe the similarity between labels by 
+ordering across one single dimension.
+For this reason we also provide the ```SubstanceParameter``` which encodes labels 
+corresponding to small molecules with chemical descriptors, capturing their similarities
+much better and without the need for the user to think about ordering and similarity at 
+all.
+This concept is generalized in the ```CustomDiscreteParameter``` where the user can 
+provide their own custom set of descriptors for each label.
+
+### ```SubstanceParameter```
+Instead of ```values```, this parameter accepts ```data``` in form of a dictionary. The 
+items correspond to pairs of labels and [SMILES](https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system).
+SMILES are a string based representation of molecular structures. 
+Based on these, BayBE can assign each label a set of molecular descriptors as encoding.
+
+For instance, a parameter corresponding to a choice of solvent can be initialized with:
+
+```python
+from baybe.parameters import SubstanceParameter
+
+SubstanceParameter(
+    name="Solvent",
+    data={
+        "Water": "O",
+        "1-Octanol": "CCCCCCCCO",
+        "Toluene": "CC1=CC=CC=C1",
+    },
+    encoding = "MORDRED", # optional
+    decorrelate = 0.7, # optional
+)
+```
+
+The ```encoding``` options define what kind of descriptors are calculated:
+* ```MORDRED```: 2D descriptors from the [Mordred package](https://mordred-descriptor.github.io/documentation/master/)
+* ```RDKIT```: 2D descriptors from the [RDKit package](https://www.rdkit.org/)
+* ```MORGAN_FP```: Morgan fingerprints calculated with RDKit (1024 bits, radius 4)
+
+These calculations will typically result in 500 to 1500 numbers per molecule.
+To avoid detrimental effects on the surrogate model fit we reduce the number of 
+descriptors before using them via decorrelation. 
+The ```decorrelate``` option in the example above specifies that only descriptors that 
+have a correlation lower than 0.7 to any other descriptor will be kept. 
+This usually reduces the number of descriptors to 10-50, depending on the specific 
+items in ```data```.
+
+```{warning}
+The descriptors calculated for a ```SubstanceParameter``` were developed to describe 
+small molecules and are not suitable for other substances. If you deal with large 
+molecules like polymers, or arbitrary substance mixtures, we recommend to provide your 
+own descriptors via the ```CustomParameter```.
+```
+
+```{warning}
+The ```SubstanceParameter``` is only available if BayBE was installed with the 
+additional ```chem``` dependency.
+```
+
+### ```CustomDiscreteParameter```
+Take the ```encoding``` concept introduced above is generalized by the 
```

Comment:
This sentence is a bit weird, please reformulate.

Reply:
fixed